index.html

```diff
@@ -0,0 +1,150 @@
+                            <div id="TextToHex" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+                                <div class="card-body">
+                                    <div class="form-group green-border-focus">
+                                        <label for="hexInput">Hex Value:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="textInput" value=""
+                                            placeholder="Text input..."></textarea>
+
+                                        <div class="row d-flex align-items-center mb-3 mt-4">
+
+
+                                            <div class="col-md-12">
+                                                <div class="text-left">
+                                                    <button type="button"
+                                                        onclick="convertText(document.getElementById('textInput').value)"
+                                                        class="btn btn-outline-success btn-rounded z-depth-1a">Convert</button>
+                                                </div>
+                                            </div>
+
+
+                                        </div>
+
+                                        <label for="textOutput">Text Output:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="hexOutput" value=""
+                                            disabled placeholder="Text Output..."></textarea>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"></script>
+    <script>
+
+        const hexInput = document.querySelector('#hexInput');
+        const textOutput = document.querySelector('#textOutput');
+        const errorMsg = document.querySelector('#errorMsg');
+        const hexInputValidation = new RegExp(/^([0-9A-Fa-f]{2})+$/g);
+
+        function convertHex(hexInput) {
+            errorMsg.innerHTML = "";
+            if(hexInput.match(hexInputValidation)){
+            let hex = hexInput.toString();
+            console.log(hex)
```

Comment:
suggest removing console logs for pull requests.

Reply:
Yes, thank you. I forgot to do so. I had it for the "debugging"